testing/test_package/lib/example.dart

```diff
@@ -357,16 +357,48 @@ class Dog implements Cat, E {
 
   /// Animation method
   ///
-  /// {@animation methodAnimation 100 100 http://host/path/to/video.mp4}
+  /// {@animation 100 100 http://host/path/to/video.mp4}
   /// More docs
   void withAnimation() {}
 
+  /// Animation method with name
+  ///
+  /// {@animation 100 100 http://host/path/to/video.mp4 id=namedAnimation}
+  /// More docs
+  void withNamedAnimation() {}
+
+  /// Animation method with quoted name
+  ///
+  /// {@animation 100 100 http://host/path/to/video.mp4 id="quotedNamedAnimation"}
+  /// {@animation 100 100 http://host/path/to/video.mp4 id='quotedNamedAnimation2'}
+  /// More docs
+  void withQuotedNamedAnimation() {}
+
+  /// Animation method with invalid name
+  ///
+  /// {@animation 100 100 http://host/path/to/video.mp4 id=2isNot-A-ValidName}
```

Comment:
Suggest adding test cases for `{@animation 250 250 id=outOfOrder http://host/path/to/video.mp4}`
and `{@animation 149 150 http://host/path/to/video.mp4 notId=unexpected}` to validate more edge case behavior

Reply:
OK, added.